src/panels/config/info/integrations-card.ts

```diff
@@ -95,8 +105,11 @@ class IntegrationsCard extends LitElement {
                                   </a>
                                 </td>
                               `
-                            : ""}
+                            : html`<td></td>`}
                         `}
+                    ${setup?.seconds
+                      ? html`<td>${setup.seconds.toFixed(2)}s</td>`
```

Comment:
```suggestion
                      ? html`<td>${setup.seconds.toFixed(2)} s</td>`
```

Just saw one thing: We usually have a blank before the unit in the frontend.